x/protocolpool/autocli.go

```diff
@@ -60,6 +60,30 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 					Example:        fmt.Sprintf(`$ %s tx protocolpool claim-budget cosmos1... --from mykey`, version.AppName),
 					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "recipient_address"}},
 				},
+				{
+					RpcMethod: "CreateContinuousFund",
+					Use:       "create-continuous-fund [title] [description] [recipient] [percentage] [cap] <metadata> <expiry>",
+					Short:     "Create continuous fund for a recipient with optional metadata and expiry",
+					Example:   fmt.Sprintf(`$ %s tx protocolpool create-continuous-fund new_title new_description cosmos1... 0.2 1000000uatom AQ== 2023-11-31T12:34:56.789Z --from mykey`, version.AppName),
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{
+						{ProtoField: "title"},
+						{ProtoField: "description"},
+						{ProtoField: "recipient"},
+						{ProtoField: "percentage"},
+						{ProtoField: "cap"},
+						{ProtoField: "metadata"},
```

Comment:
What's the use case of that? If it is because it is a gov proposal, this isn't needed. The MsgSubitProposal from gov has the metadata field, so we don't need to add it. Same goes for title, and description.